packages/@react-stately/table/src/useTableColumnResizeState.ts

```diff
@@ -5,6 +5,13 @@ import {GridNode} from '@react-types/grid';
 import {Key, MutableRefObject, useCallback, useRef, useState} from 'react';
 import {useLayoutEffect} from '@react-aria/utils';
 
+export interface AffectedColumnWidths {
```

Comment:
interface is plural but entry is single, can we change it to either be `AffectedColumnWidth` or build the array into the type?